assets/src/phoenix/keep-alive.ts

```diff
@@ -16,11 +16,15 @@
  * More Info: https://github.com/danschultzer/pow/issues/271
  * https://elixirforum.com/t/get-user-id-with-pow-from-session-for-live-view/24206/4
  */
+
+// Wait time set to 15 min which is the deafult time set by Pow to renew sessions (session_ttl_renewal)
```

Comment:
Reading pow [session expiration](https://hexdocs.pm/pow/Pow.Plug.Session.html#module-session-expiration) again, wondering if we have to set it exactly to 15 minutes, or would be safer to do it a few seconds/minute earlier so we don't hit an edge case where the session expires for not having a request in between the 15 min of `session_ttl_renewal` and the `ttl` of the session itself. Anyway, we can discuss offline if it's easier (or if I'm thinking it wrong 😄 )

Reply:
Same concern here, and I believe we should be fine with just reducing the time

Reply:
For what I read we could even set it up to 29 min and it wouldn't expire. The example it gives where it would invalidate the session in 14 minutes, is if we explicitly set up the expiration for 15 minutes. I ended up choosing 15 because is the time Pow uses to refresh, so we should be safe using the same wait time as them.
I also tried this locally setting the keep-alive to call at 25 minutes and it did work as expected.
Either way, we can move it to 10 - 14 min.